internal/content/content.go

```diff
@@ -0,0 +1,382 @@
+package content
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/go-errors/errors"
+	"github.com/google/uuid"
+	"github.com/spf13/afero"
+	"github.com/supabase/cli/internal/utils"
+	"github.com/supabase/cli/internal/utils/flags"
+	"github.com/supabase/cli/pkg/api"
+)
+
+const schemaVersion = "1"
+
+type ContentType string
+
+const (
+	ContentTypeTest    ContentType = "test"
+	ContentTypeSnippet ContentType = "snippet"
```

Comment:
we can reuse the generated enums for `api.UpsertContentBodyType` to simplify type conversion